src/3d/qgspointcloudlayer3drenderer.h

```diff
@@ -69,12 +70,18 @@ class _3D_EXPORT QgsPointCloudLayer3DRenderer : public QgsAbstract3DRenderer
     QgsPointCloudLayer3DRenderer *clone() const override SIP_FACTORY;
     Qt3DCore::QEntity *createEntity( const Qgs3DMapSettings &map ) const override SIP_SKIP;
 
+    //! Sets 3D symbol associated with the renderer
+    void setSymbol( QgsPointCloud3DSymbol *symbol );
```

Comment:
```suggestion
    /** Sets the 3D \a symbol associated with the renderer.
      * Ownership of \a symbol is transferred to the renderer.
      * \see symbol()
      */
    void setSymbol( QgsPointCloud3DSymbol *symbol SIP_TRANSFER );
```

The usual approach is for "setSymbol" type methods to take ownership of the passed object instead of internally cloning it. It's a little easier to use this way as it can potentially avoid an unwanted double clone if a caller already has an instance of the symbol which they are able to transfer. (and if not, it's trivial for the caller to call `setSymbol( symbol->clone() )`.